diff --git a/examples/meson.build b/examples/meson.build
index 62fbe9ef..53a3918e 100644
--- a/examples/meson.build
+++ b/examples/meson.build
@@ -9,35 +9,40 @@ executable(
     'telemetry-listen',
     ['telemetry-listen.c'],
     dependencies: libnvme_dep,
-    include_directories: [incdir, internal_incdir]
+    include_directories: [incdir, internal_incdir],
+    install: true
 )
 
 executable(
     'display-columnar',
     ['display-columnar.c'],
     dependencies: libnvme_dep,
-    include_directories: [incdir, internal_incdir]
+    include_directories: [incdir, internal_incdir],
+    install: true
 )
 
 executable(
     'display-tree',
     ['display-tree.c'],
     dependencies: libnvme_dep,
-    include_directories: [incdir, internal_incdir]
+    include_directories: [incdir, internal_incdir],
+    install: true
 )
 
 executable(
     'discover-loop',
     ['discover-loop.c'],
     dependencies: libnvme_dep,
-    include_directories: [incdir, internal_incdir]
+    include_directories: [incdir, internal_incdir],
+    install: true
 )
 
 executable(
     'mi-mctp',
     ['mi-mctp.c'],
     dependencies: libnvme_mi_dep,
-    include_directories: [incdir, internal_incdir]
+    include_directories: [incdir, internal_incdir],
+    install: true
 )
 
 executable(
@@ -51,7 +56,8 @@ executable(
     'mi-mctp-ae',
     ['mi-mctp-ae.c'],
     dependencies: libnvme_mi_dep,
-    include_directories: [incdir, internal_incdir]
+    include_directories: [incdir, internal_incdir],
+    install: true
 )
 
 if libdbus_dep.found()
@@ -59,6 +65,7 @@ if libdbus_dep.found()
         'mi-conf',
         ['mi-conf.c'],
         dependencies: [libnvme_mi_dep, libdbus_dep],
-        include_directories: [incdir, internal_incdir]
+        include_directories: [incdir, internal_incdir],
+        install: true
     )
 endif
